vere/sist.c

```diff
@@ -442,8 +442,8 @@ _sist_fast(u3_noun pas, c3_l key_l)
   u3_noun yek   = u3dc("scot", 'p', pas);
   c3_c*   yek_c = u3r_string(yek);
 
-  printf("saving passcode in %s/.urb/code.%s\r\n", hom_c, gum_c);
-  printf("(for real security, write it down and delete the file...)\r\n");
+  printf("boot: saving passcode in %s/.urb/code.%s\r\n", hom_c, gum_c);
+  printf("boot: for real security, write it down and delete the file\r\n");
```

Comment:
@rmariani, could you change this from "real" to "more"? This implies more than it should ...